<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="actionExport_CSV_Freepik"/>
     <addaction name="actionExport_CSV_Shutterstock"/>
     <addaction name="actionExport_CSV_Adobe_Stock"/>
     <addaction name="actionExport_CSV_iStock"/>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionOpen_Image"/>
    <addaction name="actionOpen_Multiple_Images"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_Folder"/>
    <addaction name="actionOpen_Multiple_Folders"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_Video"/>
    <addaction name="actionOpen_Multiple_Videos"/>
    <addaction name="separator"/>
    <addaction name="menuExport"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_All"/>
    <addaction name="actionDeselect_All"/>
    <addaction name="separator"/>
    <addaction name="actionRefresh"/>
    <addaction name="actionClear"/>
    <addaction name="separator"/>
    <addaction name="actionRename"/>
    <addaction name="actionRename_All"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuAI">
     <property name="title">
      <string>AI</string>
     </property>
     <addaction name="actionGoogle_Gemini"/>
     <addaction name="actionOpen_AI"/>
    </widget>
    <addaction name="actionPreferences"/>
    <addaction name="menuAI"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionWhatsApp_Group"/>
    <addaction name="separator"/>
    <addaction name="actionLicense"/>
    <addaction name="actionContributors"/>
    <addaction name="actionReport_Issue"/>
    <addaction name="actionGithub_Repository"/>
    <addaction name="separator"/>
    <addaction name="actionCheck_for_Updates"/>
    <addaction name="actionDonate"/>
    <addaction name="actionAbout_2"/>
   </widget>
   <widget class="QMenu" name="menuPrompt">
    <property name="title">
     <string>Prompt</string>
    </property>
    <addaction name="actionDefault_Prompt"/>
    <addaction name="actionCustom_Prompt"/>
    <addaction name="actionNegative_Prompt"/>
    <addaction name="actionMetadata_Prompt_2"/>
    <addaction name="separator"/>
    <addaction name="actionPrompt_Preferences"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuSettings"/>
   <addaction name="menuPrompt"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">QStatusBar{
	border-top: 1px solid rgba(114, 114, 114, 0.2); 
}</string>
   </property>
  </widget>
  <action name="actionOpen_Image">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Open Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionOpen_Multiple_Images">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Open Multiple Images</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionOpen_Folder">
   <property name="icon">
    <iconset theme="folder-open">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Open Folder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionOpen_Multiple_Folders">
   <property name="icon">
    <iconset theme="folder-open">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Open Multiple Folders</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+F</string>
   </property>
  </action>
  <action name="actionOpen_Video">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Open Video</string>
   </property>
   <property name="shortcut">
    <string>Shift+O</string>
   </property>
  </action>
  <action name="actionOpen_Multiple_Videos">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Open Multiple Videos</string>
   </property>
   <property name="shortcut">
    <string>Alt+Shift+O</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset theme="edit-copy">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset theme="edit-paste">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset theme="edit-cut">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset theme="edit-delete">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="icon">
    <iconset theme="edit-select-all">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset theme="document-new">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionExport_CSV_Freepik">
   <property name="icon">
    <iconset theme="document-export">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Export CSV Freepik</string>
   </property>
  </action>
  <action name="actionExport_CSV_Shutterstock">
   <property name="icon">
    <iconset theme="document-export">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Export CSV Shutterstock</string>
   </property>
  </action>
  <action name="actionExport_CSV_Adobe_Stock">
   <property name="icon">
    <iconset theme="document-export">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Export CSV Adobe Stock</string>
   </property>
  </action>
  <action name="actionExport_CSV_iStock">
   <property name="icon">
    <iconset theme="document-export">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Export CSV iStock</string>
   </property>
  </action>
  <action name="actionDeselect_All">
   <property name="icon">
    <iconset theme="edit-select-none">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Deselect All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+A</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset theme="view-refresh">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset theme="edit-clear">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionRename">
   <property name="icon">
    <iconset theme="edit-rename">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Rename</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionRename_All">
   <property name="icon">
    <iconset theme="edit-rename">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Rename All</string>
   </property>
   <property name="shortcut">
    <string>Shift+F2</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset theme="preferences-system">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionGoogle_Gemini">
   <property name="icon">
    <iconset theme="system-run">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Google Gemini</string>
   </property>
  </action>
  <action name="actionOpen_AI">
   <property name="icon">
    <iconset theme="system-run">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Open AI</string>
   </property>
  </action>
  <action name="actionWhatsApp_Group">
   <property name="icon">
    <iconset theme="user-group-properties">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>WhatsApp Group</string>
   </property>
  </action>
  <action name="actionLicense">
   <property name="icon">
    <iconset theme="text-x-generic">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>View License</string>
   </property>
  </action>
  <action name="actionContributors">
   <property name="icon">
    <iconset theme="system-users">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Contributors</string>
   </property>
  </action>
  <action name="actionReport_Issue">
   <property name="icon">
    <iconset theme="dialog-warning">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Report Issue</string>
   </property>
  </action>
  <action name="actionGithub_Repository">
   <property name="icon">
    <iconset theme="folder-remote">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Github Repository</string>
   </property>
  </action>
  <action name="actionCheck_for_Updates">
   <property name="icon">
    <iconset theme="software-update-available">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Check for Updates</string>
   </property>
  </action>
  <action name="actionAbout_2">
   <property name="icon">
    <iconset theme="help-about">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionDonate">
   <property name="icon">
    <iconset theme="emblem-favorite">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Donate</string>
   </property>
  </action>
  <action name="actionDefault_Prompt">
   <property name="text">
    <string>Default Prompt</string>
   </property>
  </action>
  <action name="actionCustom_Prompt">
   <property name="text">
    <string>Custom Prompt</string>
   </property>
  </action>
  <action name="actionNegative_Prompt">
   <property name="text">
    <string>Negative Prompt</string>
   </property>
  </action>
  <action name="actionMetadata_Prompt_2">
   <property name="text">
    <string>Metadata Prompt</string>
   </property>
  </action>
  <action name="actionPrompt_Preferences">
   <property name="text">
    <string>Prompt Preferences</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
